app/src/pages/inside/common/launchSuiteGrid/hamburger/hamburger.jsx

```diff
@@ -327,6 +355,25 @@ export class Hamburger extends Component {
                 customProps.onForceFinish(launch);
               }}
             />
+            {launch.retentionPolicy === RETENTION_POLICY.IMPORTANT ? (
+              <HamburgerMenuItem
+                disabled={!canUpdateImportant}
+                title={updateImportantTitle}
+                text={intl.formatMessage(messages.unmarkAsImportant)}
+                onClick={() => {
+                  this.changeImportantState();
+                }}
+              />
+            ) : (
+              <HamburgerMenuItem
+                disabled={!canUpdateImportant}
+                title={updateImportantTitle}
+                text={intl.formatMessage(messages.markAsImportant)}
+                onClick={() => {
+                  this.changeImportantState(true);
+                }}
+              />
+            )}
```

Comment:
Almost identical items that exclude each other.
Can we left just one item here?